### DISCUSSION TITLE
Error: Unexpected MODIFIER at 1, expected END

### DISCUSSION THREAD

**Discussion by fvaldes33** [vercel/community#1545] [Help] [answered]
### Summary

My Remix deployment is failing with the following error:

```
Error: Unexpected MODIFIER at 1, expected END
```

### Additional information

I am not able to verify this with a fresh install. Deploying the [remix-boilerplate](https://vercel.com/templates/remix/remix-boilerplate) works as expected. Even updating all remix dependencies to the latest 1.13.0, still works.

As I bring my minimal code over, I get the error again.

I tried using the fix mentioned in [this previous discussion](https://github.com/orgs/vercel/discussions/1282#discussioncomment-4719160), I set my VERCEL_CLI_VERSION to version of the last working deployment, 28.15.3 and it worked as expected.

Below is a copy of my package.json file.

```
{
  "private": true,
  "sideEffects": false,
  "scripts": {
    "build": "npm run build:css && remix build",
    "build:css": "tailwindcss -m -i ./styles/app.css -o app/styles/app.css",
    "dev": "concurrently \"npm run dev:css\" \"remix dev\"",
    "dev:css": "tailwindcss -w -i ./styles/app.css -o app/styles/app.css"
  },
  "dependencies": {
    "@mantine/form": "^5.10.4",
    "@remix-run/node": "^1.13.0",
    "@remix-run/react": "^1.13.0",
    "@remix-run/vercel": "^1.13.0",
    "@tabler/icons-react": "^2.4.0",
    "@vercel/analytics": "^0.1.5",
    "@vercel/node": "^2.9.5",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-hot-toast": "^2.4.0",
    "zod": "^3.20.6"
  },
  "devDependencies": {
    "@remix-run/dev": "^1.13.0",
    "@remix-run/eslint-config": "^1.13.0",
    "@remix-run/serve": "^1.13.0",
    "@tailwindcss/aspect-ratio": "^0.4.2",
    "@tailwindcss/forms": "^0.5.3",
    "@tailwindcss/typography": "^0.5.9",
    "@types/react": "^18.0.25",
    "@types/react-dom": "^18.0.9",
    "concurrently": "^7.6.0",
    "eslint": "^8.28.0",
    "tailwindcss": "^3.2.6",
    "typescript": "^4.9.3"
  },
  "engines": {
    "node": ">=14"
  }
}
```

### Edit

I slowly migrated code from my broken build to the fresh remix boilerplate repo. The breaking point was the use of a splat route in remix. Once I added that, the deployment breaks. Removing the splat route, everything works as expected.

**Comment by mikedijkstra** [comment]
My builds have also started failing. I've now set the ENV variable `VERCEL_CLI_VERSION` to `vercel@28.15.3` to get them working again. I'll follow this thread so please advise when builds are fixed and we can remove the ENV variable again.

**Reply by fvaldes33** [reply]
@mcansh has narrowed it down to here: https://github.com/vercel/vercel/blob/main/packages/remix/src/build.ts#L268-L276

I too, had to change my CLI version for now.

**Reply by mcansh** [reply]
got it working by branching on `*`

```js
const rePath = `/${path.replace(/\/\*$/, "/:params+")}`;

if (path.endsWith("*")) {
  routes.push({
    src: `/${path.replace(/\/\*$/, "/(.*)")}`,
    dest: path,
  });
} else {
  const re = (0, path_to_regexp_1.pathToRegexp)(rePath, keys);
  if (keys.length > 0) {
    routes.push({
      src: re.source,
      dest: path,
    });
  }
}
```

then `vc build && vc deploy --prebuilt`

https://vercel-splat-a21l3yqd6-mcansh.vercel.app/it/works/yay

**Reply by TooTallNate** [reply]
Sorry about this! Thanks @mcansh for helping with the repro. Fix is here: https://github.com/vercel/vercel/pull/9457

**Reply by mcansh** [reply]
jinx! https://github.com/vercel/vercel/pull/9458 - i'll close my PR

**Comment by TooTallNate** [comment, marked as answer]
Vercel CLI v28.15.7 has been released with the relevant fix. Sorry about that and thank you for the bug report!
